.github/workflows/docker-vulnerability-scan.yml

```diff
@@ -47,13 +47,13 @@ jobs:
     runs-on: ubuntu-latest
     env:
       DOCKERFILE_PATH: "ci/Dockerfile.lambda"
-      DOCKER_IMAGE: "${{ secrets.PRODUCTION_API_LAMBDA_ECR_ACCOUNT }}.dkr.ecr.ca-central-1.amazonaws.com/notify/api-lambda"
+      DOCKER_IMAGE: "${{ secrets.PRODUCTION_ACCOUNT_ID }}.dkr.ecr.ca-central-1.amazonaws.com/notify/api-lambda"
 
     steps:
       - name: Configure credentials to Notify account using OIDC
         uses: aws-actions/configure-aws-credentials@5fd3084fc36e372ff1fff382a39b10d03659f355 # v2.2.0
         with:
-          role-to-assume: arn:aws:iam::${{ secrets.PRODUCTION_API_LAMBDA_ECR_ACCOUNT }}:role/notification-api-apply
+          role-to-assume: arn:aws:iam::${{ secrets.PRODUCTION_ACCOUNT_ID }}:role/notification-api-apply
```

Comment:
This is an AWS account ID, right? is the new one a global one or specific to ECR? Can the var name be more descriptive?

Reply:
It's an AWS account ID, and it's just the corresponding environment AWS Account ID... I've been keeping it consistent across all of the other repos I've done (they've all been named ENVIRONMENT_ACCOUNT_ID) - I can change this to PRODUCTION_AWS_ACCOUNT_ID but then I should go back and change all of the others. What do you think?